tests/FSharp.Compiler.Service.Tests/FSharp.Compiler.Service.SurfaceArea.netstandard20.debug.bsl

```diff
@@ -4928,54 +4928,54 @@ FSharp.Compiler.Symbols.FSharpExpr: FSharp.Compiler.Text.Range get_Range()
 FSharp.Compiler.Symbols.FSharpExpr: Microsoft.FSharp.Collections.FSharpList`1[FSharp.Compiler.Symbols.FSharpExpr] ImmediateSubExpressions
 FSharp.Compiler.Symbols.FSharpExpr: Microsoft.FSharp.Collections.FSharpList`1[FSharp.Compiler.Symbols.FSharpExpr] get_ImmediateSubExpressions()
 FSharp.Compiler.Symbols.FSharpExpr: System.String ToString()
-FSharp.Compiler.Symbols.FSharpExprPatterns: Microsoft.FSharp.Core.FSharpOption`1[FSharp.Compiler.Symbols.FSharpExpr] |AddressOf|_|(FSharp.Compiler.Symbols.FSharpExpr)
```

Comment:
Can't change any of these APIs, they are public surface.

Reply:
Yeah I had this feeling. I reverted these particular changes